### DISCUSSION TITLE
[1161170]The Rewinder山海旅人的本体+DLC的秘钥和清单

### DISCUSSION THREAD

**Discussion by miss1ove** [pjy612/SteamManifestCache#173] [Share]
appid：1161170

key：
```
1161171;81819a3524d966ab80e641ae9eee0d3cde944eace7a7d9ce2f7afe2b9982d68e
1889430;9085dc239f31f7ed21668a5f596eeff196b269c3cc1757a70c404d3c08705f4a
```
[depotcache.zip](https://github.com/pjy612/SteamManifestCache/files/13768959/depotcache.zip)
code：681739F97EB69684
email：1802142701@qq.com

**Comment by gonghaocheng-chia** [comment]
怎么才能知道我游戏的key？

**Reply by miss1ove** [reply]
我是在Steam里面一个名称是  config.vdf   这个的文件里面找的

**Reply by gonghaocheng-chia** [reply]
![微信图片_20231226153303](https://github.com/pjy612/SteamManifestCache/assets/83695820/38d44927-d0f2-406d-939c-aa1db129d61a)
我搜236390下面没有key，是上面那个236391的吗？
